.env-example

```diff
@@ -4,3 +4,6 @@ MAILTRAP_PASSWORD=InboxPassword
 # Camo proxies insecure images in students' status updates
 CAMO_HOST=https://rgsoc-teams-camo.herokuapp.com
 CAMO_KEY=supersecret
+
+# Google Maps API key for e.g. city autocomplete
+GOOGLE_MAPS_API_KEY=YourGoogleMapsAPIKey
```

Comment:
You mean when someone copies the `.env-sample`, they want to add new keys anyway? Makes sense. I was thinking that Google Maps not working anymore could be a hidden side-effect in the event that people just want to use, say, Mailgun and copied that `.env` for that end.

But let's stick with it as it is right now!